notification/notification_test.go

```diff
@@ -0,0 +1,129 @@
+package notification_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/flanksource/commons/collections"
+	"github.com/flanksource/duty/models"
+	"github.com/flanksource/duty/types"
+	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/db"
+	dbModels "github.com/flanksource/incident-commander/db/models"
+	"github.com/flanksource/incident-commander/events"
+	"github.com/google/uuid"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = ginkgo.Describe("Test Notification on responder added", ginkgo.Ordered, func() {
+	var (
+		john      *models.Person
+		incident  *models.Incident
+		component *models.Component
+		responder *models.Responder
+		team      *dbModels.Team
+	)
+
+	ginkgo.It("should create a person", func() {
+		john = &models.Person{
+			ID:   uuid.New(),
+			Name: "James Bond",
+		}
+		tx := db.Gorm.Create(john)
+		Expect(tx.Error).To(BeNil())
+	})
+
+	ginkgo.It("should create a new component", func() {
+		component = &models.Component{
```

Comment:
I think it's good to keep it independent. Later on if any change on duty could suddenly break tests here. Example: a component name change or an incident title change on the duty dummy fixture would break this test.